app/views/items/_category_form.html.erb

```diff
@@ -0,0 +1,8 @@
+<p>
+  <%= form_for Category.new, html: { class: 'form-inline' } do |f| %>
```

Comment:
@BrandonMathis So, the `Category.new` here is the alternative to multiple variables in the controller. Is that preferable? Or is there a better way to do this?